awx/plugins/inventory/ec2.py

```diff
@@ -1201,7 +1201,7 @@ def add_elasticache_cluster(self, cluster, region):
 
         # Select the best destination address
         if 'ConfigurationEndpoint' in cluster and cluster['ConfigurationEndpoint']:
-            # Memcached cluster
+            # Redis cluster
```

Comment:
this (and below)  is actually pulling memcached clusters from AWS in inventory, it's fine to keep.

@AlanCoding is deleting this shortly anyway.

Reply:
Yep, let's leave this file in `awx/plugins` untouched in this PR, it's going away in the near future anyways.